api/.dockerignore

```diff
@@ -0,0 +1,23 @@
+**/.classpath
+**/.dockerignore
+**/.env
+**/.git

```

Comment:
This any many of these lines all start with `**/` which implies that they may be anywhere in the structure, but I don't think many actually are. I prefer that we be precise so other engineers know what we're really after. A similar analogy is something like a `select * from table_xyz` - a year later if you are trying to make changes it's hard to know the original intentions and make safe changes without unexpected side effects. The narrow scope allows future readers to make low-risk changes because they know precisely what the initial goal was.